tests/Unit/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Test_BjorhusImpl.cpp

```diff
@@ -111,12 +111,12 @@ void test_constraint_preserving_bjorhus_v_psi_vs_spec_3d(
     for (size_t a = 0; a <= VolumeDim; ++a) {
       spec_bc_dt_v_psi.get(0, a)[i] = 25.4;
     }
-    get<1, 1>(spec_bc_dt_v_psi)[i] = 31.4;
-    get<1, 2>(spec_bc_dt_v_psi)[i] = 31.4;
-    get<1, 3>(spec_bc_dt_v_psi)[i] = 31.4;
-    get<2, 2>(spec_bc_dt_v_psi)[i] = 33.4;
-    get<2, 3>(spec_bc_dt_v_psi)[i] = 33.4;
-    get<3, 3>(spec_bc_dt_v_psi)[i] = 39.4;
+    spec_bc_dt_v_psi.get(1, 1)[i] = 31.4;
```

Comment:
WHat's nvcc's problem with this? Our guideline has been to prefer the free function `get<>()` over the method `.get()` for compile-time tensor index access.

Reply:
I'm not exactly sure what sort of answer you're looking for, sorry. Most of the time it compiles the `get<>` fine, but here it doesn't. I don't remember the exact error message at this point. My guess is that the EDG parser they're using isn't as good as the GCC/LLVM ones.

Reply:
Ok I just wanted to check that we don't have to change all occurrences of this in the whole code to the method / change our guidelines